### ISSUE TITLE
Initial prisma deploy with server-side subscriptions fails

### ISSUE THREAD

**Issue by goulderb** [prismagraphql/prisma#2885]
**Describe the bug**
When performing an initial deployment of a Prisma data model, the CLI throws the following error:

```
Errors:

  Subscription
    ✖ The provided query doesn't include any known model name. Please check for the latest subscriptions API.
```

**To Reproduce**
1. Clone https://github.com/goulderb/prisma-cannot-perform-initial-deploy-with-subscriptions
2. Run docker-compose up -d
3. Run prisma deploy

**Expected behavior**
The initial deployment is performed and then the subscriptions are added after the initial deployment is complete.

**Versions (please complete the following information):**
 - OS: OS X High Sierra
 - `prisma` CLI: prisma/1.13.6 (darwin-x64) node-v10.8.0
 - Prisma Server: 1.13.6

**Additional context**
Are there any temporary workarounds that could be scripted for this issue? Right now, the only workaround that I can see from looking at the documentation is to comment out the subscriptions, run prisma deploy, uncomment the subscriptions, and run prisma deploy again. This is really unintuitive behavior.

**Comment by goulderb** [comment]
FYI, I came up with a completely hacky workaround: I have a copy of the prisma.yml file without subscriptions in another directory (initial-deploy) and an npm script that cd's into that directory, runs prisma deploy, cd's out, and runs prisma deploy again.

This isn't exactly a great solution, but it works enough to keep me limping along.

**Comment by mavilein** [comment]
I just checked the code and we indeed have a bug there. This bug can occure if one adds a new type and at the same time creates a subcription query for it. The bug is that we validate the subscription query against the old schema. So new types or other changes are not part of the schema yet.

I would recommend this work around until we have fixed this:
1. Add a new type and run `prisma deploy`.
2. Add a subscription query for this new type and run `prisma deploy`.
